### ISSUE TITLE
Strange output in footer

### ISSUE THREAD

**Issue by evictor** [jenkinsci/scm-sync-configuration-plugin#68]
I'm seeing what looks like unformatted/code output in the footer. I'm on latest Jenkins at this time, 2.138.3. I can't recall when this popped up but I'm pretty sure it was when I upgraded to 2.138.x. LMK what debug output I might be able to provide.

![image](https://user-images.githubusercontent.com/1620156/48992951-87bd9a00-f0ef-11e8-8326-b4d27d9dd709.png)

**Comment by m-bucher** [comment]
Same here on Jenkins 2.138.3 and Plugin version 0.0.10 (also tried 0.0.9)
Looks like the HTML has '<' HTML-encoded:
`&lt;br/>To remove this message, please &lt;a href='#' onclick='javascript:removeLog();'>click here&lt;/a>`

**Comment by evictor** [comment]
Bit more than cosmetic because it also covers the last bits of console output, which are often the most important. :( 

Thx for reading

**Comment by amandel** [comment]
You need to set `-Dorg.kohsuke.stapler.jelly.CustomJellyContext.escapeByDefault=false` which helps in this case. There are several things broken with this change in Jenkins. See https://jenkins.io/blog/2018/10/10/security-updates/ and https://wiki.jenkins.io/display/JENKINS/Plugins+affected+by+2018-10-10+Stapler+security+hardening

**Comment by evictor** [comment]
Thanks for the note. Is it safe to say the blanket disabling is just a band-aid and a patch should still be made so the plugin itself can handle the reconfiguration?

**Comment by mikemol** [comment]
Indeed, it would be _really_ nice if the plugin could be targeted about disabling the escaping logic.

**Comment by jeinstei** [comment]
Same issue, just to report it. It looks like the j:set var="msg" block creates a commented out span, and not what the code is expecting.

**Comment by aba-rechsteiner** [comment]
+1
